### ISSUE TITLE
Defect level 1! missing and unlinked numbers

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#22159]
Label: 18928

    - [508-defect-1 :exclamation: Launchblocker]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1

Issue Title: Defect level 1! missing and unlinked numbers
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a user, I expect that items such as phone numbers which are linked throughout a site will be linked consistantly. I expect that when asked to call a number, particularly in a crisis situation, that number will be provided and linked so I can easilly call using my smart phone.

<h2>Details</h2>
On the Health Services page, the following numbers are either missing or not linked:
<ul>
<li>Under Pharmacy, phone numbers aren’t linked.</li>
<li>Under advice nurse, the user is advised to contact an advice nurse but there is no phone number listed.</li>
<li>Under suicide prevention/veteran’s crisis line, the phone number for the crisis line isn’t linked.</li>
</ul>

Acceptance Criteria
- [ ] The phone number for the crisis line under "suicide prevention" is linked
- [ ] The phone number is provided and linked under "advice nurse"
- [ ] All phone numbers are linked under "pharmacy"

Environment

Steps to Recreate

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs

**Comment by joshkimux** [comment]
@SgardnerPMP assigning for awareness on launchblocker

**Comment by laflannery** [comment]
This is resolved.  Will close in 1 week unless told otherwise

**Comment by dsinla** [comment]
@laflannery Can you confirm you're referring to this paragraph, and the Acceptance Criteria would be that this number is linked for auto-dialing ?

![image](https://user-images.githubusercontent.com/62269210/200134367-f990b146-d66f-4fd7-aa82-6faafd08e07c.png)

**Comment by laflannery** [comment]
@dsinla yes, you can see this in the crisis line modal, how it's links as a tel: number.  I tried to do update this but I wasn't able to find this in Drupal so unfortunately I wasn't able to update this on my own
![image](https://user-images.githubusercontent.com/106678594/200343284-0439f501-91cb-4870-a8b4-2a39884bf74e.png)

**Comment by dsinla** [comment]
This text is in the VAMC description for the taxonomy term itself - not currently rich text. 
Could potentially update and see if FE picks it up? Does the <p> tag get rendered? How does the FE render it?
